src/main/java/io/spring/initializr/generator/language/Annotable.java

```diff
@@ -23,7 +23,7 @@
  *
  * @author Andy Wilkinson
  */
-public interface Annotatable {
+public interface Annotable {
```

Comment:
I think this should be `Annotatable`: https://en.oxforddictionaries.com/definition/annotatable